PlayerControls/sources/DefaultControlsViewController.swift

```diff
@@ -274,6 +282,15 @@ public final class DefaultControlsViewController: ContentControlsViewController
         onPauseEvent = visibilityController.pause
     }
     
+    func setupAirPlayCurrentVisibility() {
+        airPlayView.isHidden = !uiProps.airplayButtonHidden && airPlayView.volumeView.areWirelessRoutesAvailable
+            ? false
```

Comment:
This is not readable. Better to use `!airPlayView.volumeView.areWirelessRoutesAvailable `